hack/run-user-cluster-controller-manager.sh

```diff
@@ -36,22 +37,22 @@ if [ -z "${OWNER_EMAIL:-}" ]; then
   exit 1
 fi
 
-# We can not use the `admin-kubeconfig` secret because the user-cluster-controller-manager is
+# We cannot use the `admin-kubeconfig` secret because the user-cluster-controller-manager is
 # the one creating it in case of openshift. So we just use the internal kubeconfig and replace
-# the apiserver uurl
+# the apiserver url
 KUBECONFIG_USERCLUSTER_CONTROLLER_FILE=$(mktemp)
-ADMIN_KUBECONFIG="$(kubectl get secret internal-admin-kubeconfig -o json \
-  | jq '.data.kubeconfig' -r \
-  | base64 -d \
-  | yaml2json \
-  | jq --arg url "$CLUSTER_URL" '.clusters[0].cluster.server = $url')"
-echo $ADMIN_KUBECONFIG > $KUBECONFIG_USERCLUSTER_CONTROLLER_FILE
+kubectl --namespace "$NAMESPACE" get secret internal-admin-kubeconfig -o json | \
+  jq '.data.kubeconfig' -r | \
+  base64 -d | \
+  yq r --tojson - | \
```

Comment:
yaml2json was a tiny go program from some corner of the net, nowadays we have yq mostly everywhere.